core/src/apps/wallet/get_public_key.py

```diff
@@ -3,16 +3,38 @@
 from trezor.messages.HDNodeType import HDNodeType
 from trezor.messages.PublicKey import PublicKey
 
-from apps.common import coins, layout
+from .keychain import get_keychain_for_coin
 
+from apps.common import HARDENED, coins, layout, seed
 
-async def get_public_key(ctx, msg, keychain):
-    coin_name = msg.coin_name or "Bitcoin"
-    coin = coins.by_name(coin_name)
-    curve_name = msg.ecdsa_curve_name or coin.curve_name
+
+async def get_keychain_for_curve(ctx: wire.Context, curve_name: str) -> seed.Keychain:
+    namespaces = [
+        (curve_name, [13 | HARDENED]),
+        (curve_name, [17 | HARDENED]),
+    ]
```

Comment:
Can you add a comment explaining what these numbers 13 and 17 signify?

Reply:
fixed in 74881a23786cf1a10a74012d88a9d38c80ff69db